run.py

```diff
@@ -142,6 +142,8 @@ async def on_message(message):
     - Gets the most recent tweet from DocBohn\n
 • `BohnBot new`
     - Tells you about what is new, features, improvements, and/or commands for the most recent update\n
+• `BohnBot bohn @username`
```

Comment:
Move this to be alphabetically next (i.e. before execute). Also add this to the `README.md` to keep documentation up to date.